docs/csharp/nullable-attributes.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Upgrade APIs with attributes to define null expectations
-description: This article explains the motivations and techniques for adding descriptive attributes to describe the null state of arguments and return values from APIs
+title: Upgrade APIs for nullable reference types with attributes that define null expectations
```

Comment:
```suggestion
title: Upgrade APIs for nullable reference types with attributes that define expectations for null values
```

The previous one read kinda weird to me, but I'm not particularly intent on my specific suggestion getting in